kratos/utilities/auxiliar_model_part_utilities.h

```diff
@@ -698,6 +711,13 @@ class KRATOS_API(KRATOS_CORE) AuxiliarModelPartUtilities
         KRATOS_ERROR_IF(rContainerSize != rSize) << "mismatch in size! Expected size: " << rContainerSize << std::endl;
     }
 
+    /**
+     * @brief This method copies the submodelpart structure from the original model part to the new one.
+     * @details This method is called recursively
+     * @param rNewModelPart The new model part
+     * @param rOriginalModelPart The original model part
+     */
+    void DeepCopySubModelPart(ModelPart& rNewModelPart, const ModelPart& rOldModelPart);
```

Comment:
Is the order of the arguments (new, old) are coherent with the to from in the other methods?

Reply:
I would say yes, but I need to check

Reply:
In fact you are right https://github.com/KratosMultiphysics/Kratos/blob/2e873c3128d382503686fc26b5439f65328cc326/kratos/utilities/auxiliar_model_part_utilities.h#L110